src/app/analytics/services/analytics.service.ts

```diff
@@ -93,40 +93,33 @@ export function identifyPlan(newValue: number) {
   }
 }
 
-export function trackSignOut() {
-  /* window.analytics.track(AnalyticsTrack.SignOut);
-  window.analytics.reset(); */
+export function rudderanalyticsSignOut() {
+  window.rudderanalytics.track('User Logout');
 }
 
-export function trackSignIn(payload: { email: string; userId: string }): void {
-  // window.analytics.track(AnalyticsTrack.SignIn, payload);
+export function rudderanalyticsSignIn(uuid: string, email: string): void {
+  window.rudderanalytics.identify(uuid, { email: email, uuid: uuid }, () => {
+    console.log('Identify callback'); //For debugging
+    window.rudderanalytics.track('User Signin', { email: email });
+  });
 }
 
-export function signInAttempted(email: string, error: string | Error): void {
+export function rudderanalyticsSignInError(email: string, error: string | Error): void {
   /* window.analytics.track(AnalyticsTrack.SignInAttempted, {
     status: 'error',
     msg: error ? error : 'Login error',
     email: email,
   }); */
 }
 
-export function trackSignUp(payload: {
-  properties: { signup_source; email: string };
-  traits: {
-    member_tier?: string;
-    email: string;
-    first_name: string;
-    last_name: string;
-    usage: number;
-    createdAt: string;
-    signup_device_source: string;
-    acquisition_channel;
-  };
-  userId: string;
-}): void {
-  /* window.analytics.identify(payload.userId, payload.traits);
-  window.analytics.track(AnalyticsTrack.SignUp, payload.properties); */
-  trackSignUpServer(payload);
+export function rudderanalyticsSignUp(email: string, uuid: string): void {
+  window.rudderanalytics.identify(uuid, { email, uuid: uuid }, () => {
+    window.rudderanalytics.track('User Signup', { email });
+  });
+}
+
+export function rudderanalyticsSignUpError(email: string, error: string): void {
+  window.rudderanalytics.track('User Signup Error', { email: email, error: error });
```

Comment:
Replace error by message when sending this event, also name of the track must be `User Signup Failed`
![Screenshot 2022-11-23 at 11 08 05 a m](https://user-images.githubusercontent.com/18113007/203520230-6f2a0355-e2d0-45c8-9969-e7bc34ca481e.png)

Reply:
This function doesn't exists